tcc-transaction-unit-test/pom.xml

```diff
@@ -92,5 +92,13 @@
             <groupId>org.aspectj</groupId>
             <artifactId>aspectjweaver</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+            <version>1.6.5</version>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
There was previously no blank line between the last `</dependency>` and `</dependencies>` so probably best to leave it that way.